src/components/atoms/Badge/Badge.md

```diff
@@ -4,26 +4,38 @@ There's four different styles available:
 
 ```js
 import { Fragment } from 'react';
-import { Badge } from '@zopauk/react-components';
+import { Badge, SizedContainer } from '@zopauk/react-components';
```

Comment:
I need to remove `SizedContainer` from here as it's not used anymore on the example 💅🏻